### PR TITLE
Module-1 / linters-config

### PR DESCRIPTION
**Pull request by bilalrajput09**
### Linters Config / Hello microverse

- I am configuring the **linters** for my HTML and CSS project.
- I have used the correct **GitHub workflow**.
- I have used the correct and professional ways for **readme** file as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,90 @@
...
+
+I would like to thank Microverse for giving us the chance to build a better world.
+I would like to appreciate my team members. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./License) licensed.
```
Comment (Alfredbis29) [inline README.md]: - [ ] Good Job so far but I can see your link is broken here , kindly fix this for your project to be completed :wink:

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
### Hi @bilalrajput09 ,
 You did a great job so far 👏🏼 👏🏼 There are some issues that you still need to work on to go to the next project but you are almost there!
 
 
 ### To highlight 💪🏻
Your project is professional :white_check_mark:
Your repo is okay :white_check_mark:
No Linter errors :white_check_mark: 
:+1:  Good job!

## STATUS: CHANGES REQUESTED 
[ ]You've done a great job so far on this project, but you still need to improve on something for it to be approved. I noticed that your  `README` file the  M.I.T License is not really working since you provide a wrong link ,please include it because using the MIT License in our projects as student at Microverse can provides us  with a number of benefits, including collaboration opportunities, improved skills, portfolio building, career opportunities, and networking., kindly make sure that your M.I.T license is working properly :wink: 
Screenshot from 2023-02-15 11-12-10

 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 Cheers and Happy coding! :clap: :clap: :clap: :computer: Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please ping me @alfredbis29 when you comment so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,90 @@
...
+
+I would like to thank Microverse for giving us the chance to build a better world.
+I would like to appreciate my team members.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
+
+## 📝 License <a name="License"></a>
+
+This project is [MIT](./License) licensed.
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ]  Jut as the previous reviver mentioned,  your link is still broken here , kindly fix this for your project to be completed
- Consider the fact it's all in a capital letters on your project, then a small case on the link.

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @bilalrajput09     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements suggested. However, there are a few you need to work on, to move to the next project. 💯 



Highlights
✅All linters are checked
✅ Descriptive commit messages 
  
Required Changes ♻️

- [ ] Your MIT license file seems to be not connected based on the pat you gave it. Do review that and make sure it links well 👍 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by joel-kalema — APPROVED** [review]
Hi @bilalrajput09 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**